build/common.project.props

```diff
@@ -271,4 +271,8 @@
    <ApexProjects Include="$(RepositoryRootDirectory)test\NuGet.Tests.Apex\*\*.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-003" PrivateAssets="All" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'" />
```

Comment:
Why not `'$(TargetFramework)' == 'net46'`? 

Also you should condition on DotNetBuildFromSource as well.

Reply:
Updated for both.